runtime/model-protobuf/src/main/java/io/aklivity/zilla/runtime/model/protobuf/internal/ProtobufReadConverterHandler.java

```diff
@@ -215,6 +214,7 @@ private int validate(
                 {
                     event.validationFailure(traceId, bindingId, ex.getMessage());
                 }
+                builder.clear();
```

Comment:
Suggest we put this in a `finally` block to better document the intent, where it is required this is called after we have processed the message, agree?